bot/taskcluster-hook.json

```diff
@@ -6,82 +6,101 @@
         "name": "Code coverage hook (CHANNEL)",
```

Comment:
It'd be good to have something like https://github.com/mozilla/bugbug/blob/master/tests/test_hooks.py to validate the hook definitions.

Reply:
Agreed https://github.com/mozilla/code-coverage/issues/68